logic/src/commonMain/kotlin/com/wire/kalium/logic/data/session/SessionMapper.kt

```diff
@@ -4,31 +4,99 @@ import com.wire.kalium.logic.configuration.server.ServerConfigMapper
 import com.wire.kalium.logic.data.id.IdMapper
 import com.wire.kalium.logic.data.logout.LogoutReason
 import com.wire.kalium.logic.data.user.SsoId
-import com.wire.kalium.logic.feature.auth.AuthSession
+import com.wire.kalium.logic.feature.auth.AccountInfo
+import com.wire.kalium.logic.feature.auth.AuthTokens
 import com.wire.kalium.network.api.SessionDTO
-import com.wire.kalium.persistence.model.AuthSessionEntity
+import com.wire.kalium.persistence.client.TokenEntity
+import com.wire.kalium.persistence.dao_kalium_db.AccountInfoEntity
+import com.wire.kalium.persistence.model.SsoIdEntity
+import com.wire.kalium.persistence.model.LogoutReason as LogoutReasonEntity
 
 interface SessionMapper {
-    fun toSessionDTO(authSession: AuthSession.Session.Valid): SessionDTO
-    fun fromSessionDTO(sessionDTO: SessionDTO): AuthSession.Session.Valid
-
-    fun fromPersistenceSession(authSessionEntity: AuthSessionEntity): AuthSession
-    fun toPersistenceSession(authSession: AuthSession, ssoId: SsoId?): AuthSessionEntity
+    fun toSessionDTO(authSession: AuthTokens): SessionDTO
+    fun fromEntityToSessionDTO(tokenEntity: TokenEntity): SessionDTO
+    fun fromSessionDTO(sessionDTO: SessionDTO): AuthTokens
+    fun fromAccountInfoEntity(accountInfoEntity: AccountInfoEntity): AccountInfo
+    fun toLogoutReasonEntity(reason: LogoutReason): LogoutReasonEntity
+    fun toSsoIdEntity(ssoId: SsoId?): SsoIdEntity?
+    fun toAuthTokensEntity(authSession: AuthTokens): TokenEntity
+    fun fromSsoIdEntity(ssoIdEntity: SsoIdEntity?): SsoId?
+    fun toLogoutReason(reason: LogoutReasonEntity): LogoutReason
+    // fun fromPersistenceSession(authSessionEntity: AuthSessionEntity): AuthSession
+    // fun toPersistenceSession(authSession: AuthSession, ssoId: SsoId?): AuthSessionEntity
```

Comment:
Are we planning to bring these again or should we remove them?

Reply:
these are gone no need to expose AuthTokens to the app 🚀